src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -956,6 +957,7 @@ namespace
             create_pt_advanced_nee_dl_settings(layout);
             create_pt_advanced_nee_ibl_settings(layout);
             create_pt_advanced_nee_max_ray_intensity_settings(layout);
+            create_pt_advanced_nee_clamp_roughness_settings(layout);
```

Comment:
I'm not sure Clamp Roughness should go inside NEE. I think that even without NEE, Clamp Roughness is effective. Can you confirm?

Reply:
Sorry, I'm not sure about that. May be @est77 can answer?

Reply:
What I meant is this: Is roughness clamping working when you disable NEE?

Reply:
I'm going to merge this as it looks good, and we can move Roughness Clamping outside of NEE params afterward.